app/components/summary_card/style.scss

```diff
@@ -0,0 +1,100 @@
+@function frontend-font-url($filename) {
```

Comment:
how would these functions be shared with other components which may need them?

Reply:
Good point, this was duplicated between the normal application.scss and this component. 

We have abstracted this setup into a different file called `govuk_style_setup.scss`.